src/modules/sched-simple/sched.c

```diff
@@ -330,6 +330,46 @@ static void cancel_cb (flux_t *h,
     }
 }
 
+/* Job manager indicates there is a priority change to a job.  If a
+ * matching job found in queue, update the priority and reorder queue
+ * as necessary.
+ */
+void prioritize_cb (flux_t *h, flux_msg_handler_t *mh,
+                    const flux_msg_t *msg, void *arg)
+{
+    struct simple_sched *ss = arg;
+    json_t *jobs;
+    size_t index;
+    json_t *arr;
+
+    if (flux_request_unpack (msg, NULL, "{s:o}", "jobs", &jobs) < 0)
+        goto proto_error;
+
+    json_array_foreach (jobs, index, arr) {
+        flux_jobid_t id;
+        unsigned int priority;
+        json_int_t tmp;
```

Comment:
change type of priority to `int64_t` and drop tmp variabl

Reply:
ahhh got it, keep `unsigned int priority` or `int priority` in the primary job structs, but not in locals.